Automattic/wp-calypso
# Update help center handling of external links

Links from support articles that we show in the help center were opened on the same page.
This PR introduces a click handler which prevents the default behavior for links and opens them in a new tab.

#### Proposed Changes

* Parse article content links to open them in new tab

#### Testing Instructions
1. Navigate to the new help center
2. Open an Article. Example `What are "Blocks"?`
3. Click on some of the article's external links
4. Confirm that the links are opened in a new tab

Related to #64676
